lib/wine_o_inventory_api/inventories/inventory.ex

```diff
@@ -0,0 +1,17 @@
+defmodule WineOInventoryApi.Inventories.Inventory do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "inventories" do
+    belongs_to :member, WineOInventoryApi.Users.Member
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(inventory, attrs) do
+    inventory
+    |> cast(attrs, [])
+    |> validate_required([])
```

Comment:
If we aren't validating anything, is this required?

Reply:
No, it is not! I'll go ahead and remove it.